browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -201,8 +210,11 @@ function _treelistboxEntry(parentContainer, treeViewData, entry, builder, isTree
 
 	var expander = L.DomUtil.create('div', builder.options.cssClass + ' ui-treeview-expander ', span);
 
-	if (entry.state !== undefined)
-		var selectionElement = _createSelectionElement(span, treeViewData, entry,builder);
+	var selectionElement;
+	if (entry.image)
+		selectionElement = _createImageColumn(parentContainer, builder, entry);
```

Comment:
please put it in the columns processing loop - also look at the core patch comment wrt that